_includes/work-for-us.html

```diff
@@ -54,7 +54,7 @@
             <li>3 weeks paid vacation</li><br>
             <li>Year end paid break</li><br>
             <li>Competitive medical and dental benefits fully funded by Greenfield</li><br>
-            <li>We bill clients weekly for a 4 day work week, so Fridays are devoted to personal skill development and exploration</li><br>
+            <li>We bill clients weekly for a 4 day work week, Fridays are devoted to personal skill development and exploration</li><br>
```

Comment:
Grammatically, it could be either a semicolon or a period, but I think a period is fine here.

Also "workweek" is one word 👍